selfdrive/car/hyundai/values.py

```diff
@@ -447,6 +449,33 @@ def match_fw_to_car_fuzzy(live_fw_versions, offline_fw_versions) -> Set[str]:
       obd_multiplexing=False,
     ),
 
+    # CAN & CAN FD logging queries (from camera)
+    Request(
+      [HYUNDAI_VERSION_REQUEST_LONG],
+      [HYUNDAI_VERSION_RESPONSE],
+      whitelist_ecus=ALL_HYUNDAI_ECUS,
+      bus=0,
+      auxiliary=True,
+      logging=True,
+    ),
+    Request(
+      [HYUNDAI_VERSION_REQUEST_MULTI],
+      [HYUNDAI_VERSION_RESPONSE],
```

Comment:
Not running this on bus 1 PT since it's too many queries and not likely to get anything. If we see interesting ECUs on HDA1 cars on bus 0/4 that only respond to multi, we can add it for HDA2's bus 1